.devcontainer/devcontainer.json

```
@@ -1,10 +1,7 @@
 {
-  "service": "frontend-dev",
-  "dockerComposeFile": "../docker-compose.yaml",
-  "forwardPorts": [3000],
-  "customizations": {
-    "vscode": {
-      "extensions": ["GitHub.copilot"]
-    }
+  "name": "Open-Assistant",
+  "image": "mcr.microsoft.com/vscode/devcontainers/universal",
+  "features": {
+    "ghcr.io/devcontainers-contrib/features/pre-commit:2": {"version": "2.0.1"}
```

```suggestion
    "ghcr.io/devcontainers-contrib/features/pre-commit:2": {
      "version": "2.0.1"
    }
```

my bad, might need be something like this to make pre-commit happy

---

can see it complaining here: https://github.com/LAION-AI/Open-Assistant/actions/runs/3830383352/jobs/6518170592

---

@andrewm4894 
Also when I tried to run it, it gives me the following error:
```[ERROR] The hook `black` requires pre-commit version 2.9.2 but version 2.0.1 is installed.  Perhaps run `pip install --upgrade pre-commit`.```

I'll try to change the version and report back with that.